continuous_integration/circle/build_doc.sh

```diff
@@ -16,7 +16,7 @@ conda update --yes --quiet conda
 conda create -n $CONDA_ENV_NAME --yes --quiet python=3
 source activate $CONDA_ENV_NAME
 
-conda install --yes --quiet pip numpy sphinx matplotlib pillow
+conda install --yes --quiet pip numpy sphinx matplotlib pillow dask distributed
```

Comment:
Just `dask` suffices here.  The core package is now called `dask-core` while `dask` is a metapackage that includes `distributed` and a few other packages (like numpy, pandas, ...)